models/cloudtemplate_region_detail.go

```diff
@@ -12,30 +12,30 @@ const (
 	CloudtemplateRegionDetailClassName = "cloudtemplateRegionDetail"
 )
 
-type Additionalconfigforregion struct {
+type CloudTemplateRegion struct {
 	BaseAttributes
-	AdditionalconfigforregionAttributes
+	CloudTemplateRegionAttributes
 }
 
-type AdditionalconfigforregionAttributes struct {
+type CloudTemplateRegionAttributes struct {
 	Annotation           string `json:",omitempty"`
 	HubNetworkingEnabled string `json:",omitempty"`
 }
 
-func NewAdditionalconfigforregion(cloudtemplateRegionDetailRn, parentDn string, cloudtemplateRegionDetailAttr AdditionalconfigforregionAttributes) *Additionalconfigforregion {
+func NewCloudTemplateRegion(cloudtemplateRegionDetailRn, parentDn string, cloudtemplateRegionDetailAttr CloudTemplateRegionAttributes) *CloudTemplateRegion {
 	dn := fmt.Sprintf("%s/%s", parentDn, cloudtemplateRegionDetailRn)
```

Comment:
do we need this cloudtemplateRegionDetailRn  passed as argument when it is static value?

Reply:
ok